libbeat/cloudid/cloudid_test.go

```diff
@@ -79,11 +79,11 @@ func TestDecode(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		esURL, kbURL, err := decodeCloudID(test.cloudID)
+		cid, err := NewCloudID(test.cloudID, "")
 		assert.NoError(t, err, test.cloudID)
 
-		assert.Equal(t, esURL, test.expectedEsURL, test.cloudID)
-		assert.Equal(t, kbURL, test.expectedKibanaURL, test.cloudID)
+		assert.Equal(t, cid.esURL, test.expectedEsURL, test.cloudID)
+		assert.Equal(t, cid.kibURL, test.expectedKibanaURL, test.cloudID)
```

Comment:
Better prefer blackbox testing: `cid.ElasticsearchURL` over checking internals like `cid.esURL`.

Reply:
Good catch, I forgot to update the tests after implementing the getter methods. Thanks!